api_v3/services/UserEntryService.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @service userEntry
+ * @package api
+ * @subpackage services
+ */
+class UserEntryService extends KalturaBaseService {
```

Comment:
It is not needed according to design and in the meantime during integration use it hasn't been requested by anyone